kas/include/base.yml

```diff
@@ -0,0 +1,50 @@
+header:
+  version: 20
+
+distro: "yoe"
+
+repos:
+  yoe-distro:
+    path: .
+    layers:
+      sources/meta-yoe:
+
+  openembedded-core:
+    url: https://github.com/YoeDistro/openembedded-core
+    branch: master
+    path: layers/openembedded-core
+    layers:
+      meta:
+
+  bitbake:
+    url: https://github.com/YoeDistro/bitbake
+    branch: master
+    path: layers/bitbake
+    layers:
+      bitbake: disabled
+
+  meta-openembedded:
```

Comment:
This looks like more than the 'base'. In my opinion the base.yml should only clone bitbake, and layers that are listed in `LAYERDEPENDS_meta-yoe` so basically only openembedded-core.
All other layers (marked as dynamic layer inside meta-yoe) should be in a separate yaml file that other yaml files can include based on there needs.

Reply:
Hi @lslrt

Something like this? https://github.com/riscv/meta-riscv/blob/master/kas/include/oe.yml